ocs_ci/utility/utils.py

```diff
@@ -954,6 +954,34 @@ def ocsci_log_path():
     )
 
 
+def get_testrun_id():
```

Comment:
Should we rename this to `get_polarion_testrun_id` as to not confuse the return value from this with `config['RUN']['run_id']`?

Reply:
I was thinking about the `polarion` prefix, but it is used also for email and html reports... what do you thing about `get_testrun_name` instead?

Reply:
I think `get_testrun_name` makes sense

Reply:
Fixed by 065f186.